civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -263,3 +263,116 @@
     }
   }
 }
+
+.page-features-in-civihr {
+  .onboarding_wizard_features_get_started {
+    background: $crm-wizard-secondary-color;
+    display: block;
+    margin: 40px auto 0 auto;
+    text-transform: uppercase;
+    width: 200px;
+  }
+
+  .page-header {
+    font-weight: 700;
+    margin-top: 80px;
+    text-align: center;
+
+    .title {
+      font-weight: 500;
+    }
+  }
+
+  .view-id-onboarding_slideshow {
+    background: transparent;
+  }
+
+  .views-slideshow-cycle-main-frame {
+    margin: 0 30px;
+    height: 303px;
+
+    .views-slideshow-cycle-main-frame-row {
+      width: 100% !important;
+
+      img {
+        height: 303px !important;
+        width: 458px !important;
+      }
+
+      .views-slideshow-cycle-main-frame-row-item {
+        margin: 0 65px;
+        position: relative;
+
+        .views-field-field-slideshow-image {
+          box-shadow: 0 10px 20px 0 rgba(63,60,85,0.3), 0 -2px 15px 0 rgba(63,60,85,0.3);
```

Comment:
In sass you can use `rgba($color, 0.3)` or `rgba(#000000, 0.3)`, can you use those instead? It's difficult to determine which color are these.

Reply:
Done